.github/workflows/pr.yml

```
@@ -39,3 +39,13 @@ jobs:
           skip: "*.svg,*.js,*.map,*.css,*.scss"
           ignore_words_list: "aks,atleast,cros,fiel,ist,ot,pullrequest,ser,shttp"
           path: docs
+    deploy-to-staging:
+      uses: SumoLogic/sumologic-documentation/.github/workflows/build_and_deploy.yml@main
```

Didn't you just change the name of the build_and_deploy.yml file to `build-and-deploy.yml`, with hyphes instead of underscores? :thinking: 
```suggestion
      uses: SumoLogic/sumologic-documentation/.github/workflows/build-and-deploy.yml@main
```

---

This section was copied directly from the deploy-to-staging file https://github.com/SumoLogic/sumologic-documentation/blob/main/.github/workflows/main.yml